docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,9 +150,42 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Setting and Getting Playback Capability
+
+To get playback capability from the client and to set it on the server side, follow these steps:
+
+1. To set playback capability on the server side, use the `SetPlaybackCapability` method of the  [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
+
+    ```csharp
+    MediaControlServer.SetPlaybackCapability(MediaControlPlaybackCommand.FastForward, MediaControlCapabilitySupport.NotSupported);
+    ```
+
+2. To get playback capability on the client side, use the `GetPlaybackCapability` method of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
+
+    ```csharp
+    var playCommandCapability = controller.GetPlaybackCapability((MediaControlPlaybackCommand.Play);
```

Comment:
Would you please check doubled "("? 
((MediaControlPlaybackCommand.Play)

Reply:
Done